moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -528,14 +534,14 @@ const NewProjectMap = ({
                 )}
               />
             </Source>
-          )
-        )}
+          );
+        })}
 
         {/* Street Tool Tip*/}
         {renderTooltip(featureText, hoveredCoords, classes.toolTip)}
 
         {/* Draw tools */}
-        {renderMapDrawTools(mapEditToolsContainerRef)}
+        {!isSignalComponent && renderMapDrawTools(mapEditToolsContainerRef)}
```

Comment:
Similar to above, we don't want users hand-drawing geometries for signal components.